start.js

```diff
@@ -0,0 +1,20 @@
+var spawn = require('child_process').spawn
+var os = require('os')
+var start
+
+function onLog (data) {
+  console.log(data.toString())
+}
+
+function onError (data) {
+  console.error(data.toString())
+}
+
+if (os.type() === 'Linux') {
```

Comment:
Sadly I have to inform you that this isn't working on macOS. `require('os').type() === 'Darwin'` on macOS.

```suggestion
if (os.type() === 'Windows_NT') {
  start = spawn('node', ['bin\\start.js'])
} else {
  start = spawn('./bin/start.js')
}
```